patches/tModLoader/Terraria/ID/NPCID.cs.patch

```diff
@@ -17,3 +17,228 @@
  			public static Dictionary<int, int> SpecialSpawningRules = new Dictionary<int, int> {
  				{ 259, 0 },
  				{ 260, 0 },
+@@ -1153,7 +_,7 @@
+ 
+ 				dictionary.Add(6, value);
+ 				value = new NPCBestiaryDrawModifiers(0) {
+-					CustomTexturePath = "Images/UI/Bestiary/NPCs/NPC_7",
++					CustomTexturePath = "Terraria/UI/Bestiary/NPCs/NPC_7",
```

Comment:
This is far from ideal, please find another solution, even if it means some hardcode string manipulation where `CustomTexturePath` is used, and a `TODO` message

Reply:
This is the only way that I could get it to actually work with the current state of the asset system. If I just did a string.Replace thing turning Images/ into Terraria/, then it would also affect mods where im certain some will have an images folder they put everything in, which doesnt really make it feasible. I do not plan for this to final. I'll definitely add a comment.

Reply:
`if (CustomTexturePath.StartsWith("Images/")) CustomTexturePath = "Terraria" + CustomTexturePath.SubString(6);`

works unless a mod is named images, since mod name has to come first